server/.env

```diff
@@ -1,3 +1,13 @@
+<<<<<<< Updated upstream
 TEAM_NAME="Shums, Jhanvi, Simeon"
 FLASK_ENV="development"
 
+=======
```

Comment:
oops merge conflict

Reply:
ignore this `.env` file in `.gitignore` 

this is essentially pushing out the credentials, make sure amazon doesn't ping you or to be safe, change your pswd and change this key as well to avoid any attacks